// [[Reason: Implicit undefined in parameter can only be detected by TSC.]] ////

//// [tests/cases/compiler/declarationEmitParameterProperty.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,22 +1,7 @@
 
 
 //// [declarationEmitParameterProperty.d.ts]
 export declare class Foo {
-    bar?: invalid;
-    constructor(bar?: invalid);
+    bar?: string;
+    constructor(bar?: string);
 }
-
-/// [Errors] ////
-
-declarationEmitParameterProperty.ts(2,15): error TS9025: Declaration emit for this parameter requires implicitly adding undefined to it's type. This is not supported with --isolatedDeclarations.
-
-
-==== declarationEmitParameterProperty.ts (1 errors) ====
-    export class Foo {
-      constructor(public bar?: string) {
-                  ~~~~~~~~~~~~~~~~~~~
-!!! error TS9025: Declaration emit for this parameter requires implicitly adding undefined to it's type. This is not supported with --isolatedDeclarations.
-!!! related TS9028 declarationEmitParameterProperty.ts:2:15: Add a type annotation to the parameter bar.
-      }
\ No newline at end of file
-    }
-    
